services/workflows-service/scripts/workflows/full_kyb_process.ts

```diff
@@ -0,0 +1,310 @@
+      //   on: {
+      //     CONTINUE: [{ target: 'run_ubos' }],
+      //     // @TODO: add 404 handling
+      //     FAILED: [{ target: 'auto_reject' }],
+      //   },
+      // },
+      // run_ubos: {
+      //   tags: [StateTag.PENDING_PROCESS],
+      //   on: {
+      //     CONTINUE: [{ target: 'run_kyb_enrichment' }],
+      //     FAILED: [{ target: 'auto_reject' }],
+      //   },
+      // },
+      manual_review: {
+        tags: [StateTag.MANUAL_REVIEW],
+        on: {
+          approve: 'approved',
+          reject: 'rejected',
+          revision: 'revision',
+        },
+      },
+      revision: {
+        tags: [StateTag.REVISION],
+        always: [
+          {
+            target: 'pending_resubmission',
+          },
+        ],
+      },
+      pending_resubmission: {
+        tags: [StateTag.REVISION],
+        on: {
+          RESUBMITTED: 'manual_review',
+        },
+      },
+      approved: {
+        tags: [StateTag.APPROVED],
+        type: 'final' as const,
+      },
+      rejected: {
+        tags: [StateTag.REJECTED],
+        type: 'final' as const,
+      },
+      auto_reject: {
+        tags: [StateTag.REJECTED],
+        type: 'final' as const,
+      },
+    },
+  },
+  extensions: {
+    apiPlugins: [
+      {
+        name: 'asia_verify_cn_kyb',
+        pluginKind: 'kyb',
+        url: `${env.UNIFIED_API_URL}/companies-v2`,
+        method: 'GET',
+        stateNames: ['run_cn_kyb'],
+        successAction: 'CN_KYB_DONE',
+        errorAction: 'CN_KYB_FAIL',
+        headers: { Authorization: 'Bearer {secret.UNIFIED_API_TOKEN}' },
+        request: {
+          transform: [
+            {
+              // @TODO: Work out "entity.data.registrationNumber" with Daniel
+              transformer: 'jmespath',
+              mapping: `{
+                companyNumber: entity.data.registrationNumber,
+                countryOfIncorporation: 'cn',
+                vendor: 'asia-verify'
+              }`, // jmespath
+            },
+          ],
+        },
+        response: {
+          transform: [
+            {
+              transformer: 'jmespath',
+              mapping: '@', // jmespath
+            },
+          ],
+        },
+      },
+      {
+        // @TODO: handle callback
+        name: 'asia_verify_hk_kyb',
+        pluginKind: 'webhook',
+        url: `${env.UNIFIED_API_URL}/companies-v2`,
+        method: 'GET',
+        stateNames: ['run_hk_kyb'],
+        headers: { Authorization: 'Bearer {secret.UNIFIED_API_TOKEN}' },
+        request: {
+          transform: [
+            {
+              // @TODO: Work out "entity.data.registrationNumber" with Daniel
+              transformer: 'jmespath',
+              mapping: `{
+                companyNumber: entity.data.registrationNumber,
+                countryOfIncorporation: 'hk',
+                vendor: 'asia-verify'
+              }`, // jmespath
+            },
+          ],
+        },
+        response: {
```

Comment:
Currently, we are not using the response (its the persistence is done from /hooks endpoint)